packages/crayons-styles/src/index.js

```diff
@@ -0,0 +1,25 @@
+import crayons from '../dist/design-tokens/crayons.json';
+import { convertTokenToString } from './utils';
+
+function setStyle(tokens) {
+  const cssString = convertTokenToString(tokens);
+  const sheet = new CSSStyleSheet();
+  sheet.replaceSync(`:root {${cssString}}`);
+  document.adoptedStyleSheets = [sheet];
+}
+
+export function setTheme(theme) {
+  if (theme === 'crayons') {
+    setStyle(crayons);
+  }
+  import(`../dist/design-tokens/${theme}.json`)
```

Comment:
point to a different folder other than dist here.

Reply:
Renamed the output folder of the design tokens